sessions/react-data-fetching/star-wars/components/Card/index.js

```diff
@@ -0,0 +1,16 @@
+export default function Card({ name, height, eyeColor, birthYear }) {
+  return (
+    <div>
```

Comment:
We could use a styled component adding a border or a background here to make it appear like a card ;-)